packages/web/src/pages/collection-page/components/desktop/CollectionPage.tsx

```diff
@@ -352,6 +352,11 @@ const CollectionPage = ({
                   isAlbum ? messages.type.album : messages.type.playlist
                 }`}
                 isAlbumPage={isAlbum}
+                isAlbumPremium={
+                  !!metadata && 'is_stream_gated' in metadata
```

Comment:
`metadata?.is_stream_gated ?? false`

Reply:
`metadata` could be a `SmartCollection` so we have to check 😢